catroid/res/values/strings.xml

```diff
@@ -285,9 +287,10 @@
     <!-- New sprite dialog -->
     <string name="new_sprite_dialog_default_sprite_name">My Object</string>
     <string name="new_sprite_dialog_title">New Object</string>
-    <string name="spritename_already_exists">This object name already exists</string>
+    <string name="spritename_already_exists">Please choose another object name.</string>
     <string name="rename_sprite_dialog">Rename Object</string>
     <string name="spritename_invalid">This object name is invalid</string>
+    <string name="no_spritename_entered">Please enter a object name.</string>
```

Comment:
This applies to other nouns like look, sound, image, brick, script, backpack and even more.
I'll write an UX ticket for that tomorrow.